st2auth/st2auth/backends/__init__.py

```diff
@@ -14,23 +14,31 @@
 # limitations under the License.
 
 import json
-import importlib
 
 from oslo_config import cfg
+from stevedore.driver import DriverManager
+from stevedore.extension import ExtensionManager
 
-from st2common.util.loader import _get_classes_in_module
+from st2common import log as logging
 
 __all__ = [
-    'get_backend_instance',
-    'VALID_BACKEND_NAMES'
+    'get_available_backends',
+    'get_backend_instance'
 ]
 
-BACKEND_MODULES = {
-    'flat_file': 'st2auth.backends.flat_file',
-    'mongodb': 'st2auth.backends.mongodb'
-}
+LOG = logging.getLogger(__name__)
 
-VALID_BACKEND_NAMES = BACKEND_MODULES.keys()
+BACKENDS_NAMESPACE = 'st2auth.backends.backend'
```

Comment:
It's just a "convention" I followed from the stevedore documentation examples. it's a backend which belongs to "st2auth.backends" package, hence st2auth.backends.backend.

In any case, it's not a big deal so either st2auth.backend or st2auth.backends works.